### ISSUE TITLE
Clarification: does "information about the user" imply that the information is current and accurate?

### ISSUE THREAD

**Issue by brothercake** [w3c/wcag#3542]
1.3.5 Identify Input Purpose speaks of "information about the user", which is then ratified in [Input Purposes for User Interface Components](https://www.w3.org/TR/WCAG22/#input-purposes) that speaks of input control purposes that "relate to the user of the content and pertain only to information related to that individual".

Does this imply that the information is current and accurate (e.g., the user's current name, and their current postal address), or does it also include old information (e.g., their previous name if they've changed it, or a postal address they no longer live at, both of which might be requested in a credit application for example)?

If old information is in scope, does the SC need to be concerned with the fact that using the same token in multiple fields is not practically useful for browser autocomplete behavior (e.g., in a form where current and previous names are both requested, using `given-name` on both fields), or would this imply that we need tokens to identify old values explicitly?

Alternatively, if it isn't in scope, does this need to be clarified, that the SC only includes current and accurate information?

**Comment by alastc** [comment]
Task force answer: 
<hr>

The SC itself doesn't define this, but the purposes (as defined in section 7) are intended to be for current valid info, not old info. However, if people add an input purpose to fields for old information, that is not a fail (as the SC doesn't require NOT using it).

**Comment by giacomo-petri** [comment]
I think we can close this. 
The author of the issue has accepted Alastair’s suggestion. 

TF to confirm the resolution for acknowledgment (I don't think updating the SC is needed but to discuss in TF meeting)
